src/Pretzel.Logic/Import/WordpressImport.cs

```diff
@@ -66,7 +66,12 @@ private void ImportPost(WordpressPost p)
             var postsFolder = "_posts";
             var fileName = string.Format("{0}-{1}.md", p.Published.ToString("yyyy-MM-dd"), p.PostName.Replace(' ', '-')); //not sure about post name
 
-            fileSystem.File.WriteAllText(Path.Combine(pathToSite, postsFolder, fileName), postContent);
+            var path = Path.Combine(pathToSite, postsFolder, fileName);
```

Comment:
Same here, we should avoid the call to Path.GetDirectoryName:
``` cs
var postsPath = Path.Combine(pathToSite, "_posts");
var path = Path.Combine(postsPath, fileName);

if(!fileSystem.Directory.Exists(postsPath))
...
```

Reply:
But the directory doe's not exist so it failes now. It's a behavior change. `WriteAllText` failes if the directory does not exist.

Reply:
But we can optimize the directory name creation.

Reply:
I did get you wrong. Never mind 👍

Reply:
Fixed.